VargasMartinezJose/Dia2/HackerRank/ejerciciojava.java/JavaStdinStdout1.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class JavaStdinStdout1{
+
+    public static void main(String[] args) {
+        
+        Scanner scan = new Scanner(System.in);
+       
+
+        int integer_a = scan.nextInt();
+        int integer_b = scan.nextInt();
+        int integer_c = scan.nextInt();
```

Comment:
variables in java are written with lowerCamelCase, Don't use _
put a descriptive name